deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+
+COPY 1.3.0/manifests /manifests/
+COPY 1.3.0/metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```

Comment:
does this mean 1.3.0 was available on ocp 4.6?
is 1.4.0 available for ocp 4.4?

Reply:
- 1.3.0 is no longer available on ocp 4.6, the first publish has been done by red-hat team was wrong but it was fixed 
- yes 1.4.0 available for ocp 4.4, the new bundle project is relevant from version 4.5 (4.4 is published by the old way..)

Reply:
got it! thanks